.gitignore

```diff
@@ -107,4 +107,4 @@ tmp/
 .texlipse
 
 # kotlin
-annotations/
+/annotations/
```

Comment:
@lazaroclapp this originally led to our new `com.uber.nullaway.annotations` package source files getting ignored.  Is this the right fix?

Reply:
No idea. Do we have Kotlin code in this repo?

Reply:
I don't think so 🙂 Should I just remove the line then?

Reply:
Let's try that :)

Reply:
done :-)